### PR TITLE
Bullet changes

### PR DESCRIPTION
**Pull request by TheBucketBip**
- Fixes  some of the 10mm ammo types, which were previously unusable.

- Adds a new ammo type for 10mm called "rubber" which is a non-lethal variant of 10mm. 

- The armor penetration on hollow point has been lowered from 12 to 8.

- Made it so hollow point does more damage than normal 10mm rounds, otherwise it's just a worse version of 10mm.

- Made it so the armor penetration on AP rounds for 10mm is 25 instead of 24.

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -143,16 +143,24 @@
...
 	agony = 31
 
-/obj/item/projectile/bullet/pistol/medium/smg/hollowpoint
+/obj/item/projectile/bullet/pistol/medium/smg/rubber
 	fire_sound = 'sound/weapons/gunshot/gunshot_smg.ogg'
-	damage = 34 //10mm hollowpoint
+	damage = 1.5 //10mm rubber
 	armor_penetration = 12
-	agony = 25
+	agony = 57
```
Comment (rhicora) [inline code/modules/projectiles/projectile/bullets.dm]: This is an enormous amount of agony damage per shot and keep in mind that smgs can be fired in burst mode

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -143,16 +143,24 @@
...
+	embed = 0
+	sharp = 0
+
+/obj/item/projectile/bullet/pistol/medium/smg/hollowpoint
+	fire_sound = 'sound/weapons/gunshot/gunshot_smg.ogg'
+	damage = 46 //10mm hollowpoint
```
Comment (rhicora) [inline code/modules/projectiles/projectile/bullets.dm]: There isn't much of a reason to buff hollowpoints, they're already absurd at 34.

File: code/modules/projectiles/projectile/bullets.dm [hunk 3]
```diff
@@ -143,16 +143,24 @@
...
+	armor_penetration = 8
+	agony = 35
```
Comment (rhicora) [inline code/modules/projectiles/projectile/bullets.dm]: There isn't much of a reason to buff hollowpoint agony either, 25 was high.

**Comment by TheBucketBip** [comment]
Yeah, I thought hollow points were suppose to do more damage than normal ammo at the cost of armor penetration, didn't notice the agony part. I'll go fix that, don't remember making the agony higher for hollow points though. I forgot about the burst fire mode so i'll modify the agony on rubber bullets as well.

**Review by rhicora — APPROVED** [review]
Code itself looks fine now, needs Lion to check whether he wants agony ammo to exist for SMGs.

**Comment by JakeDaBoss** [comment]
@rhicora smg's are agony themselves. I'm good with this, lmao. Could make for some good new ammo types that we can use, or be given by default.

**Pull request merged**